statsmodels/tsa/arima_model.py

```diff
@@ -925,7 +927,10 @@ def fit(self, start_params=None, trend='c', method="css-mle",
             self.nobs = len(self.endog) - k_ar
 
         if start_params is not None:
-            start_params = np.asarray(start_params)
+            if hasattr(start_params, '__call__'):
```

Comment:
The [documentation for callable()](https://docs.python.org/3/library/functions.html#callable]) says
>New in version 3.2: This function was first removed in Python 3.0 and then brought back in Python 3.2.